assets/src/design-system/icons/chevron_up_small.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M12 17L16 14L20 17" stroke="#E4E5E6" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M12 17L16 14L20 17" stroke="currentColor" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
FWIW our SVGO config should be doing this automatically under the hood.

Good practice nontheless 👍

Reply:
Might this be missing for Storybook only then? Because without it, the icons clearly weren't colorable - which is why I noticed these exact problems.

Reply:
Strange.

We definitely do load svgr in our storybook webpack config:

https://github.com/google/web-stories-wp/blob/38e3513c7e8ba1d46c5bde5d5572c662d89803cc/.storybook/main.cjs#L52

If that's broken, would you mind opening a ticket for us to investigate? Thanks!

Reply:
I can confirm it doesn't work in Storybook, only in the actually editor build. Maybe it's because `svgo.yml` is in the root directory, so it's not picked up inside the `storybook` folder? Follow-up ticket in #6357.